pkg/utils/utils.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+func EnsureLxcfsParentDir(path string) error {
+	lxcDir := filepath.Dir(strings.TrimRight(path, "/"))
+	if !strings.HasSuffix(lxcDir, "lxc") {
```

Comment:
I normally use `/var/lib/lxcfs-on-kubernetes` to avoid any potential conflict with lxcfs running in the node by other means like LXC.

Reply:
@felipecrs yeah. It's a good idea